ts/parts-map/HeatmapSeries.ts

```diff
@@ -18,6 +18,15 @@ import H from '../parts/Globals.js';
  */
 declare global {
     namespace Highcharts {
+        type HeatmapSeriesStatesInactiveExceptValue = (
+            'row'|'column'|'column-row'|'self'|'all'
+        );
+
+        type HeatmapInactiveFilterFunction = (
```

Comment:
Please make function types as interfaces, because type declarations are just placeholder which results in messy tooltips when it comes to functions. In your case:
```
interface HeatmapInactiveFilterFunction {
    (activePoint: HeatmapPoint, otherPoint: HeatmapPoint): boolean;
}
```

Reply:
Ok, will update, thanks!